README.md

```diff
@@ -57,6 +57,52 @@ You probably have enabled "Secure Window" feature in SealNote settings.
 Disable it to be able to take screenshots or show window content in application
 switcher.
 
+Contributing
+--------------------------------
+
+Contributions are much appreciated. Please go through this section to understand how and what to contribute.
+
+If you need ideas for contribution, please check currently [open issues](https://github.com/vishesh/sealnote/issues). If you want to propose a new feature, please open a new issue. A good feature can be something which improves some workflow or security for a user.
+
+Before you start working on an issue, please claim it so that there is no duplication of effort. We don't assign an issue until you have a passing pull-request.
+
+### Tools of trade
+
+1. Java 1.7+
+2. Android API Level 19+
+
+### How to submit a pull request
+
+1. Clone sealnote:
+    `git clone https://github.com/vishesh/sealnote.git && cd sealnote`
```

Comment:
If you want to put it in a block, there is no need to wrap it inside "`". Just indentation is fine. "`" is for inline code.